### PR TITLE
Handled issues, enabled fonts/button customization for us to use

### PR DESCRIPTION
**Pull request by andreavieira**
Will continue adding UI for app, add option for inviting friends, improve home screen and end fragment, and (stretch) nest fragments for player/mode selection on same screen, so that the display will be nice in portrait and landscape mode.

### CODE REVIEW

File: app/src/main/res/layout/activity_home_screen.xml [hunk 1]
```diff
@@ -8,32 +8,29 @@
...
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_alignParentBottom="true"
         android:layout_centerHorizontal="true"
-        android:layout_marginBottom="177dp"
-        android:text="Log Out"
-        tools:layout_editor_absoluteX="148dp"
-        tools:layout_editor_absoluteY="373dp" />
+        android:text="@string/logout_btn"
+        android:paddingVertical="30dp" />
```
Comment (samhithak) [inline app/src/main/res/layout/activity_home_screen.xml]: Move to dimens.xml

File: app/src/main/java/a3/com/convo/fragments/ModeFragment.java [hunk 2]
```diff
@@ -28,19 +31,53 @@ public ModeFragment() {
...
+        final TextView tvPickTime = (TextView) view.findViewById(R.id.tvPickTime);
+        final EditText timeInput = (EditText) view.findViewById(R.id.etPickTime);
+        final Button playButton = (Button) view.findViewById(R.id.playButton);
+
         startGame = (Button) view.findViewById(R.id.start_game_btn);
         startGame.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                ((PlayGameActivity)context).goToGame(friend);
+                tvPickTime.setText(getString(R.string.pick_game_time));
```
Comment (samhithak) [inline app/src/main/java/a3/com/convo/fragments/ModeFragment.java]: check if tvPickTime is null

File: app/src/main/java/a3/com/convo/fragments/ModeFragment.java [hunk 3]
```diff
@@ -28,19 +31,53 @@ public ModeFragment() {
...
+                            ((PlayGameActivity) getContext()).goToGame(friend, Constants.FREESTYLE, seconds);
+                        }
+                    }
+                });
+                layout.setVisibility(View.VISIBLE);
+            }
+        });
+
+        Button timedMode = (Button) view.findViewById(R.id.timed_mode);
+        timedMode.setOnClickListener(new View.OnClickListener() {
```
Comment (samhithak) [inline app/src/main/java/a3/com/convo/fragments/ModeFragment.java]: check if timedMode is null

File: app/src/main/java/a3/com/convo/fragments/ModeFragment.java [hunk 4]
```diff
@@ -28,19 +31,53 @@ public ModeFragment() {
...
+            @Override
+            public void onClick(View view) {
+                tvPickTime.setText(getString(R.string.pick_card_time));
+                playButton.setOnClickListener(new View.OnClickListener() {
```
Comment (samhithak) [inline app/src/main/java/a3/com/convo/fragments/ModeFragment.java]: Why do we have onclick listener inside another?

File: app/src/main/java/a3/com/convo/fragments/ModeFragment.java [hunk 5]
```diff
@@ -28,19 +31,53 @@ public ModeFragment() {
...
+                });
+                layout.setVisibility(View.VISIBLE);
+            }
+        });
+
+        Button timedMode = (Button) view.findViewById(R.id.timed_mode);
+        timedMode.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                tvPickTime.setText(getString(R.string.pick_card_time));
```
Comment (samhithak) [inline app/src/main/java/a3/com/convo/fragments/ModeFragment.java]: Can we add screenshots to this PR, helps me understand how it looks

File: app/src/main/java/a3/com/convo/activities/ProfileActivity.java [hunk 6]
```diff
@@ -11,10 +11,6 @@
 import a3.com.convo.fragments.ProfileDetailsFragment;
 
 public class ProfileActivity extends AppCompatActivity {
-
-    private AddInfoFragment addInfoFrag;
-    private ProfileDetailsFragment profileDetailsFrag;
-    private Fragment profileFrag;
     private FragmentTransaction ft;
```
Comment (samhithak) [inline app/src/main/java/a3/com/convo/activities/ProfileActivity.java]: Make this local variable?

File: app/src/main/java/a3/com/convo/activities/PlayGameActivity.java [hunk 7]
```diff
@@ -15,7 +14,6 @@
 import a3.com.convo.fragments.ModeFragment;
 
 public class PlayGameActivity extends AppCompatActivity {
-
     private FragmentTransaction ft;
```
Comment (samhithak) [inline app/src/main/java/a3/com/convo/activities/PlayGameActivity.java]: Make this local variable

File: app/src/main/java/a3/com/convo/activities/ProfileActivity.java (hunk 6 above)
Comment (andreavieira) [reply app/src/main/java/a3/com/convo/activities/ProfileActivity.java]: This is no longer in my code, I must have selected older code on accident during merge.

File: app/src/main/java/a3/com/convo/fragments/FriendsFragment.java [hunk 8]
```diff
@@ -44,19 +37,20 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
+        RecyclerView friendsRv = view.findViewById(R.id.rv_friends);
+        final FriendAdapter friendAdapter = new FriendAdapter(friends);
+        friendsRv.setLayoutManager(new LinearLayoutManager(getActivity()));
+        friendsRv.setAdapter(friendAdapter);
 
         startButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                ((PlayGameActivity)context).goToMode(friendAdapter.getSelectedFriend());
+                Log.d("---->", "onClicked");
```
Comment (samhithak) [inline app/src/main/java/a3/com/convo/fragments/FriendsFragment.java]: Remove this

File: app/src/main/java/a3/com/convo/activities/ProfileActivity.java (hunk 6 above)
Comment (samhithak) [reply app/src/main/java/a3/com/convo/activities/ProfileActivity.java]: Can you rebase and push

**Pull request closed without merging**